.github/workflows/nightly-benchmark.yml

```diff
@@ -83,12 +85,16 @@ jobs:
           git config user.name 'github-actions[bot]'
           git config user.email 'github-actions[bot]@users.noreply.github.com'
           git commit -m "Update benchmark results"
+
+          git push origin "$branch_name"
+
           # create a new PR
           gh pr create \
             --title "chore(gas_costs): update local chain config with nightly benchmark results" \
             --body "Updated benchmark results" \
             --base master \
             --head "$branch_name"
```

Comment:
This is missing a `\` at the end of this line to include the `--label` parameter.

Reply:
yes we have an issue to fix this